src/main/java/com/sequenceiq/cloudbreak/service/stack/resource/gcc/builders/instance/GccInstanceResourceBuilder.java

```diff
@@ -204,6 +216,80 @@ public ResourceType resourceType() {
         return ResourceType.GCC_INSTANCE;
     }
 
+    @Override
+    public Boolean stop(GccStartStopContextObject startStopContextObject, Resource resource, String region) {
+        GccCredential credential = (GccCredential) startStopContextObject.getStack().getCredential();
+        try {
+            Compute.Instances.Stop stop = startStopContextObject.getCompute().instances()
+                    .stop(credential.getProjectId(), GccZone.valueOf(region).getValue(), resource.getResourceName());
+            stop.setPrettyPrint(Boolean.TRUE);
+            Operation execute = stop.execute();
+            if (execute.getHttpErrorStatusCode() == null) {
+                Compute.ZoneOperations.Get zoneOperations = createZoneOperations(
+                        startStopContextObject.getCompute(),
+                        credential,
+                        execute,
+                        GccZone.valueOf(startStopContextObject.getStack().getRegion()));
+                GccResourceReadyPollerObject instReady = new GccResourceReadyPollerObject(zoneOperations, startStopContextObject.getStack(),
+                        resource.getResourceName(), execute.getName(), ResourceType.GCC_INSTANCE);
+                gccInstanceReadyPollerObjectPollingService.pollWithTimeout(gccResourceCheckerStatus, instReady, POLLING_INTERVAL, MAX_POLLING_ATTEMPTS);
+                return true;
+            } else {
+                return false;
+            }
+        } catch (IOException e) {
+            LOGGER.error(String.format("There was an error in the vm stop [%s]: %s", resource.getResourceName(), e.getMessage()));
+            return false;
+        }
+    }
+
+    @Override
+    public Boolean start(GccStartStopContextObject startStopContextObject, Resource resource, String region) {
```

Comment:
start and stop is almost the same. instead of duplicating most of the code we could use one method with a boolean stop parameter like in the AzureVirtualMachineResourceBuilder